### DISCUSSION TITLE
Add search integration to docs?

### DISCUSSION THREAD

**Discussion by thdoan** [withastro/docs#3150] [Ideas]
I like how the Astro team implemented their search functionality on https://docs.astro.build/ -- it would be cool to see a guide on how to do the same on https://docs.astro.build/en/guides/integrations-guide/ :)

UPDATE: I found these two articles, but not sure if it's the same implementation as on the official docs site:

https://blog.otterlord.dev/post/astro-search/
https://danidiaztech.com/create-astro-search-component/

**Comment by delucis** [comment]
On the docs site we use Algolia DocSearch: https://docsearch.algolia.com/

That programme is limited to open source projects so I'd be hesitant to document it here. We do link to the first blog post you found in our community recipes section here: https://docs.astro.build/en/recipes/#community-resources

**Comment by ArmandPhilippot** [comment]
We're closing stale and outdated discussion to keep only the discussions still relevant to the current project version and/or the current docs state.

If you think this is a mistake and that this discussion should remain open, please feel free to ping me, I'll take a second look at it.

Thank your for your contribution!

And in case someone lands here, the docs now uses Starlight and the [Search feature](https://starlight.astro.build/guides/site-search/) is documented in Starlight docs!
